### PR TITLE
Rank corr

### PR DESCRIPTION
**Pull request by tweddielin**
- Add react-faux-dom in package.json (https://github.com/Olical/react-faux-dom) which is the best way to leverage both react.js and d3.js I found so far. 
- Create a heat-map for rank-correlation or jaccard smiliarity
- Chang some chart components' names for better understanding

### CODE REVIEW

File: frontend/components/rankcorr-chart/component.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      .attr("y", function(d, i) {
+        return (i+1) * cellSize + 50
+      })
+      .style("text-anchor", "end")
+      .attr("transform", "translate(80," + cellSize / 1.5 + ")")
+      .attr("class", function(d, i) {
+        return "rowLabel mono r" + i
+      })
+
+    const colLabels = svg.append("g")
```
Comment (thcrock) [inline frontend/components/rankcorr-chart/component.js]: Agreeing with the hound on colLabels and rowLabels. Do these variables need to exist or can the transformations just be done and a comment added at the top of each section to give you context about them?

File: frontend/package.json [hunk 2]
```diff
@@ -30,10 +30,12 @@
     "react": "^15.4.1",
     "react-datepicker": "~0.39.0",
     "react-dom": "^15.4.1",
+    "react-faux-dom": "^3.0.0",
     "react-nvd3": "^0.5.7",
     "react-table": "^3.1.4",
     "react-tabs": "~0.8.2",
-    "reactable": "~0.14.1"
+    "reactable": "~0.14.1",
+    "underscore": "^1.8.3"
```
Comment (thcrock) [inline frontend/package.json]: Is underscore still needed? I don't see it used. Even if it was, for any underscore operation I could probably show you a ramda equivalent, but from the looks of it we don't have to cross that bridge here.

File: frontend/components/rankcorr-chart/component.js (hunk 1 above)
Comment (tweddielin) [inline frontend/components/rankcorr-chart/component.js]: They actually used in the graph.

File: frontend/package.json (hunk 2 above)
Comment (tweddielin) [inline frontend/package.json]: - They actually used in the graph.
- Removed the underscore.

**Pull request merged**